keying.go

```diff
@@ -0,0 +1,47 @@
+package srtp
+
+// KeyingMaterialExporter allows package SRTP to extract keying material
+type KeyingMaterialExporter interface {
+	// TODO: To match crypto/tls label should be a string.
+	ExportKeyingMaterial(label []byte, context []byte, length int) ([]byte, error)
+}
+
+// ExtractSessionKeysFromDTLS extracts the session keys from DTLS
+// This behavior is defined in RFC5764
+// https://tools.ietf.org/html/rfc5764
+func ExtractSessionKeysFromDTLS(exporter KeyingMaterialExporter, isClient bool) (SessionKeys, error) {
```

Comment:
We need `ProtectionProfile` in here somewhere, the length/offsets are determine by that (and we will need to support that in the future) don't want to break the API when that happens.